specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2017-07-01/bms.json

```diff
@@ -157,6 +157,58 @@
       }
     },
     "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupFabrics/{fabricName}/backupProtectionIntent/{intentObjectName}": {
+      "get": {
+        "tags": [
+          "ProtectionIntent"
+        ],
+        "description": "Provides the details of the protection intent up item. This is an asynchronous operation. To know the status of the operation,\r\n            call the GetItemOperationResult API.",
+        "operationId": "ProtectionIntent_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "Fabric name associated with the backed up item.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "intentObjectName",
+            "in": "path",
+            "description": "Backed up item name whose details are to be fetched.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Add a "default" response to capture all the error states. the schema complies with the ARM error contract in RPC. Check Batch RP swagger for sample

Reply:
Thanks for pointing this out @vladimirjoanovic Looks like this is missing in multiple places. We will add the default response to all the API's in our spec in next iteration. I created an issue to track this. URL - https://github.com/Azure/azure-rest-api-specs/issues/3989 

Can you please resolve this comment?

Reply:
Please add 'default' per Vlad's comment to the new APIs. Your task should be used to retroactively fix existing issues.

Reply:
Hi Ryan. We depend on SwashBuckle utilities to generate the Swagger spec. Currently, there's no attribute to add the default tag. We will create a custom attribute and fix it.

Reply:
Product team agreed to fix 'default' across all API's in the next iteration